docs/changelog.rst

```diff
@@ -123,6 +129,14 @@ Cleanups
 - Catch errors when loading :confval:`logging/config_file`.
   (Fixes: :issue:`1320`)
 
+Audio
+-----
+
+- **Breaking:** The audio scanner now returns ISO-8601 formatted strings
```

Comment:
We should be all good then. Mopidy-Spotify is the only extension that *needs* a new release to work with Mopidy 1.2 (which in semver should be named 2.0, but, well, we control the only thing that is broken).